repos/keg-core/core/base/hooks/store/useStoreItems.js

```diff
@@ -0,0 +1,75 @@
+import { useSelector, shallowEqual } from 'react-redux'
+import { isStr, get, capitalize } from '@keg-hub/jsutils'
+
+/**
+ * Helper hook for getting categories from the store.
+ * Particularly useful if you are gathering multiple categories at once.
+ * @param {*} categories
+ * @param {*} comparisonFn - used to compare if data changed
+ * @return {Object<string, (Object|Array)> | (Object|Array)} an object containing the categories requested, if they exist in the store, or the category items themselves (returned only if `categories` is a path string)
+ * @example
+ * const { todos, users, settingsAgenda } = useStoreItems(['todos', 'users', 'settings.agenda'])
+ *
+ * @example
+ * const names = useStoreItems('users.names')
+ */
+export const useStoreItems = (categories = [], comparisonFn = shallowEqual) => {
+  const isSingleCategory = isStr(categories)
+  return useSelector(
+    store => isSingleCategory
+      ? selectPath(store.items, categories)
+      : selectCategories(store.items, categories),
+    comparisonFn
+  )
+}
+
+/**
+ * selectPath: gets the path from items, so long as both exist and path is a string
+ * @param {object} items 
+ * @param {string} path 
+ * @return {*?} - the value stored in items at path
+ */
+const selectPath = (items, path) => {
```

Comment:
This method seems kind of point less. If you need this, update jsutils `updateColl` method to check it the path is an array or string.
Then on line 49 just call `get` like this => `get(items, path, null)`

Reply:
agreed, removed. It can just be a plain get call since it returns defined when path is empty/null.